dashboard/config/scripts/levels/AnjaliTest Sprite Lab Project.level

```diff
@@ -0,0 +1,88 @@
+    "auto_run_setup": "DRAW_LOOP",
+    "parent_level_id": 4578,
```

Comment:
Did you create this level as a new level, or as a clone? I _think_ this makes it a template-backed level (shares code with the parent level) but I'm not sure it's clearly documented anywhere, except for in the code.

Reply:
> LGTM! Are you planning to prevent playing sounds in live-preview in a follow-on PR?

Yeah, working on that now.

Reply:
Yeah, I cloned a level. Fixed to make a standalone level.